tgg/research/gnomad_tgps.py

```diff
@@ -87,7 +118,7 @@ def get_pangenome_ids(
     ids = []
     with hfs.open(id_path, "r") as file:
         for line in file:
-            sid = line.split(" ")[1].strip()
+            sid = line.split(PANGENOME_ID_DELIM)[PAGENOME_ID_COL_NUM].strip()
```

Comment:
this works here, but in the future, probably good to do this if you want defaults:
```
def get_pangenome_ids(
    id_path: str,
    delim: str = PANGENOME_ID_DELIM,
    col_num: int = PANGENOME_ID_COL_NUM,
) -> hl.expr.SetExpression:
```

Reply:
also this suggestion is for your reference; no need to add this to the script if you do not want to